### DISCUSSION TITLE
Change GPU reading ROG Ally

### DISCUSSION THREAD

**Discussion by Palladrian** [seerge/g-helper#3325] [General]
Hello, is it possible to change the settings where ghelper gets the gpu readings from?

**Comment by seerge** [comment]
@Palladrian hello, what exactly do you mean?

**Reply by Palladrian** [reply]
@seerge  Hello, I opened a thread some weeks ago beginning September #3074 and now I am wondering if it is possible to change it so the GPU temps are going to be the same as in AMD Adrenaline.

**Reply by seerge** [reply]
@Palladrian it's possible, but what is the practical reason for this?. Cause current temp that you see - is the one BIOS uses to control "GPU" fan. Most likely AMD driver just reports something like GPU hotpoint temp or so (so it's not average temp on the APU chip, but the hottest point)

**Reply by Palladrian** [reply]
@seerge I realised that when armory crate is installed the gpu and cpu temps are the same. Meanwhile ghelper can be even 20-30 degrees off. To make it short it would make controlling the fan speed much easier for me personally as the cpu fan can get up to 6000 while the gpu sits at 3000rpm because the reported temp is lower. I know I can set higher fan speeds for lower temps but would like to know how can I change where ghelper gets the GPU reading from

**Reply by seerge** [reply]
@Palladrian but G-Helper is not controlling your fans. They are controlled by BIOS along internal algorithm. So it does not matter where app would get temp readings, it won't affect actual fan behavior.

So temperature reported in the app would have only informational / cosmetic effect :)
